### PR TITLE
Bug Fix: label service_id for gateway metrics

### PR DESCRIPTION
**Pull request by GLYASAI**
1. missing label serivcie for gateway metrics
2. add ssl_protocols TLSv1.1 TLSv1.2 TLSv1.3

### CODE REVIEW

File: hack/contrib/docker/gateway/nginxtmp/servers.tmpl [hunk 1]
```diff
@@ -14,6 +14,7 @@ server {
 
     {{ if .SSLCertificate }}
     # ssl
+    ssl_protocols TLSv1.1 TLSv1.2 TLSv1.3;
```
Comment (barnettZQG) [inline hack/contrib/docker/gateway/nginxtmp/servers.tmpl:17]: 1. What is the OpenSSL version? Whether to support TLSv1.3?
2. Should a parameter be set to the configuration, which in some cases can be set by environment variables?

Comment (barnettZQG) [reply hack/contrib/docker/gateway/nginxtmp/servers.tmpl:17]: ingress-controller default values is "TLSv1.2 TLSv1.3"

**Comment by barnettZQG** [comment]
The gateway non-component metric should be filtered in the available metric query api.

**Comment by GLYASAI** [comment]
> The gateway non-component metric should be filtered in the available metric query api.

https://github.com/goodrain/rainbond/pull/892#issuecomment-735733294

File: hack/contrib/docker/gateway/nginxtmp/servers.tmpl (hunk 1 above)
Comment (GLYASAI) [reply hack/contrib/docker/gateway/nginxtmp/servers.tmpl]: fixed

**Review by barnettZQG — APPROVED** [review]

**Pull request merged**